TestMod/README.md

```diff
@@ -0,0 +1,2 @@
+# TestMod
```

Comment:
We might want to consider splitting out TestMod into it's own repo since it's more an example/demo later on. Microsoft does it this way with React Native where the RN core is it's own repo only but the examples are stored in another.